src/react/components/common/keyboardManager/keyboardRegistrationManager.test.ts

```diff
@@ -33,63 +33,67 @@ describe("Keyboard Registration Manager", () => {
     });
 
     it("can register handlers for same key code and different key event types", () => {
-        const keyCode = "Ctrl+H";
+        const keyCodeString = "Ctrl+H";
+        const keyCodes = [keyCodeString];
```

Comment:
Do you need both of these? Could you just use keyCode and [keyCode]?

Reply:
I could use [keyCode]